### PR TITLE
Fix some of doxygen warnings.

### PR DESCRIPTION
**Pull request by endJunction**
Mainly correct variable names in the documentation.
Remove some documentation for non-existing parameters.

### CODE REVIEW

**Comment by endJunction** [comment]
Jenkins, test this please!

**Comment by endJunction** [comment]
Jenkins, test this please!

**Comment by endJunction** [comment]
@bilke Any ideas why that particular docker job fails with same errors?

**Comment by TomFischer** [comment]
Jenkins, test this please.

File: NumLib/Fem/Integration/IntegrationGaussPrism.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ class IntegrationGaussPrism
      * get coordinates of a integration point
      *
      * @param order    the number of integration points
-     * @param pt_id     the sampling point id
+     * @param ipg      the sampling point id
```
Comment (TomFischer) [inline NumLib/Fem/Integration/IntegrationGaussPrism.h]: Paramter is `igp`, documented parameter is ipg, i.e., switch pg.

File: NumLib/Fem/Integration/IntegrationGaussPyramid.h [hunk 2]
```diff
@@ -62,7 +62,7 @@ class IntegrationGaussPyramid
      * get coordinates of a integration point
      *
      * @param order    the number of integration points
-     * @param pt_id     the sampling point id
+     * @param ipg      the sampling point id
```
Comment (TomFischer) [inline NumLib/Fem/Integration/IntegrationGaussPyramid.h]: Same as above.

File: NumLib/Fem/Integration/IntegrationGaussTri.h [hunk 3]
```diff
@@ -78,7 +78,7 @@ class IntegrationGaussTri
      * get coordinates of a integration point
      *
      * @param order    the number of integration points
-     * @param pt_id     the sampling point id
+     * @param ipg      the sampling point id
```
Comment (TomFischer) [inline NumLib/Fem/Integration/IntegrationGaussTri.h]: Same as above.

**Comment by chleh** [comment]
:+1:

**Pull request merged**